lib/IGameCallback.cpp

```diff
@@ -47,11 +47,11 @@ void CPrivilagedInfoCallback::getFreeTiles (std::vector<int3> &tiles) const
 	}
 }
 
-void CPrivilagedInfoCallback::getTilesInRange( std::unordered_set<int3, ShashInt3> &tiles, int3 pos, int radious, boost::optional<PlayerColor> player/*=uninit*/, int mode/*=0*/, bool patrolDistance/*=false*/) const
+void CPrivilagedInfoCallback::getTilesInRange(std::unordered_set<int3, ShashInt3> &tiles, int3 pos, int radious, boost::optional<PlayerColor> player/*=uninit*/, int mode/*=0*/, int distanceFormula/*=0*/) const

```

Comment:
I understand why we have unordered here (FoW calculation), but in most cases vector<int3> is enough and much faster. It would be good to have getTilesInRange with vector container too.